packages/cdk8s-cli/templates/python-app/.hooks.sscaff.js

```diff
@@ -14,14 +14,35 @@ exports.pre = () => {
 };
 
 exports.post = options => {
-  const pypi_cdk8s = options.pypi_cdk8s;
+  const { pypi_cdk8s, pypi_cdk8s_plus } = options;
   if (!pypi_cdk8s) {
     throw new Error(`missing context "pypi_cdk8s"`);
   }
+  if (!pypi_cdk8s_plus) {
+    throw new Error(`missing context "pypi_cdk8s_plus"`);
+  }
 
   execSync('pipenv lock --clear')
-  execSync('pipenv install', { stdio: 'inherit' });
+  execSync('pipenv install --skip-lock', { stdio: 'inherit' });
   execSync(`pipenv install ${pypi_cdk8s}`, { stdio: 'inherit' });
+  /**
+   * Using --no-deps flag here to ignore subdependencies. For cdk8s_plus, that's
```

Comment:
Was there a continuation for this sentence? :)

Reply:
Was meaning to list the dependencies. Updated the wording!